### PR TITLE
Add Coop

### PR DESCRIPTION
**Pull request by jokay**
Adds [Coop](https://www.coop.ch/en/).

rank-coop

### CODE REVIEW

File: _data/retail.yml [hunk 1]
```diff
@@ -110,6 +110,13 @@ websites:
     img: comixology.png
     twitter: comiXology
 
+  - name: Coop
+    url: https://www.coop.ch
+    img: coop.png
+    twitter: coop_ch
+    facebook: coop.ch
+    lang: de
```
Comment (phallobst) [inline _data/retail.yml:118]:
```suggestion
    lang: de
    regions:
      - ch
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Comment by Carlgo11** [comment]
I'm a bit hesitant about the name. There are lots of grocery chains in Europe named Coop (that don't seem to be affiliated in any way)

Could we use something like `Coop (Switzerland)`?

**Comment by jokay** [comment]
> I'm a bit hesitant about the name. There are lots of grocery chains in Europe named Coop (that don't seem to be affiliated in any way)
> 
> Could we use something like `Coop (Switzerland)`?

Or we could use `Coop Genossenschaft` like it's written on the [imprint](https://www.coop.ch/de/unternehmen/impressum.html)?

**Comment by Carlgo11** [comment]
Sure!

**Review by Carlgo11 — APPROVED** [review]

**Comment by jokay** [comment]
Changed accordingly 👍🏼

**Pull request merged**